src/sentry/search/snuba/executors.py

```diff
@@ -164,19 +164,23 @@ def _convert_search_filters(
                     "environment": environments,
                 },
             )
-            # if no re-formatted conditions, use fallback method
-            converted_filters.append(
-                conditions[0]
-                if conditions
-                else convert_search_filter_to_snuba_query(
+
+            # if no re-formatted conditions, use fallback method for selected groups
+            new_condition = None
+            if conditions:
+                new_condition = conditions[0]
+            elif group_ids:
+                new_condition = convert_search_filter_to_snuba_query(
                     search_filter,
                     params={
                         "organization_id": organization_id,
                         "project_id": project_ids,
-                        "environment": environments,
+                        "environment_id": environments,
```

Comment:
I think this has to be `environment`:
https://github.com/getsentry/sentry/blob/8e45ea7afa1b96de300f33fc92cd7f772639c5ff/src/sentry/search/events/filter.py#L573

Reply:
oops didn't mean to change, copy paste error